### PR TITLE
Modification of footer

### PR DESCRIPTION
**Pull request by sj056**
John Doe _ Home - Google Chrome 2021-01-08 19-50-00 (1)
John Doe _ Home - Google Chrome 2021-01-08 19-51-53 (1)
John Doe _ Home - Google Chrome 2021-01-08 19-53-46

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Don't Recommend to use grey here instead use a white and black colour combination or a lighter shade of black for this

**Comment by smaranjitghose** [comment]
# Feedback:
1.  Please do not change the color of the navigation bar from black to gray. Gray does not go with white at all!

navvv
^^ The toggle button is poorly messed up. Please be cautious of the changes you do

2. The color of the social media icons in the home page should not be changed

3. Reduce the size of the contact form. It is bloating out a lot! 😅

4. Remove that white line between the contact form and the left side of the footer

5. The send button is never the size of the input fields

File: research.html [hunk 1]
```diff
@@ -32,14 +32,18 @@
...
   <!--Import materialize.css-->
   <link type="text/css" rel="stylesheet" href="assets/css/materialize.min.css" media="screen,projection" />
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300&display=swap" rel="stylesheet">
   <!--Import animation.css-->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" crossorigin="anonymous" />
   <script src='https://kit.fontawesome.com/a076d05399.js'></script>
   <link rel="stylesheet" type="text/css" href="assets/css/research.css" />
   <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+  <link rel="stylesheet" href="assets/css/home.css">
```
Comment (smaranjitghose) [inline research.html:45]: This is not to be called for every page. We have a style.css for this purpose. Please understand the importance of code structure

File: skill.html [hunk 2]
```diff
@@ -41,8 +42,11 @@
   <!--Import animation.css-->
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css" />
   <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css">
   <link rel="stylesheet" type="text/css" href="assets/css/project.css" />
   <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+  <link rel="stylesheet" href="assets/css/home.css"> 
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
```
Comment (smaranjitghose) [inline skill.html:49]: I would always suggest local assets for icons

File: skill.html [hunk 3]
```diff
@@ -32,6 +32,7 @@
     content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" crossorigin="anonymous" />
```
Comment (smaranjitghose) [inline skill.html:35]: Why are there two versions of font-awesome?

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: research.html (hunk 1 above)
Comment (sj056) [reply research.html:45]: Sir the style properties for social media icons were defined in home.css due to that all the pages except index.html couldn't load the footer properly.  That's why I have added home.css in all templates else I have to remove that code from home.css and add that in style.css .

Comment (smaranjitghose) [reply research.html:45]: Yes. Please shift it to the style.css

File: skill.html (hunk 3 above)
Comment (sj056) [reply skill.html:35]: > # Feedback:
> 1. Please do not change the color of the navigation bar from black to gray. Gray does not go with white at all!
> 
> navvv
> ^^ The toggle button is poorly messed up. Please be cautious of the changes you do
> 
> 1. The color of the social media icons in the home page should not be changed
> 2. Reduce the size of the contact form. It is bloating out a lot! 😅
> 3. Remove that white line between the contact form and the left side of the footer
> 4. The send button is never the size of the input fields

>Here are the changes :

Screenshot (210)

Screenshot (211)

Comment (sj056) [reply skill.html:35]: > Why are there two versions of font-awesome?

Sorry for that I have removed the older version now.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Refer this PR for colour codes rest the design is all right https://github.com/smaranjitghose/awesome-portfolio-websites/pull/449

**Comment by Abhijay007** [comment]
To resolve the above conflicts you need to pull upstream master.  use git version control for that, 
you can also follow the steps mentioned in the readme file.

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**